### ISSUE TITLE
Remove outdated tasks

### ISSUE THREAD

**Issue by DavertMik** [consolidation/robo#1161]
![image](https://github.com/consolidation/robo/assets/220264/4c133bd6-81e0-4cc4-a80c-d6c937982420)

Most of those tasks were introduced to build frontend assets, however, they are not used in a modern world. We can safely remove them in next major version

* Gulp
* Bower
* Assets
*

**Comment by ivangretsky** [comment]
I do not think that those (maybe except for Bower) should be removed altogether, A lot of projects still use those. Maybe move them to 1st party plugins instead and create a plugin page on the website?
